File: loj/3/solution.cs
using System;
using System.IO;
using System.Linq;

class Program
{
    static void Main()
    {
        if (!File.Exists("copycat.in"))
        {
            File.WriteAllText("copycat.out", string.Empty);
            return;
        }

        var tokens = File.ReadAllText("copycat.in")
            .Split((char[])null, StringSplitOptions.RemoveEmptyEntries);

        if (tokens.Length == 0)
        {
            File.WriteAllText("copycat.out", string.Empty);
            return;
        }

        int t = int.TryParse(tokens[0], out var n) ? n : 0;
        var vals = tokens.Skip(1).Take(t);
        File.WriteAllText("copycat.out", string.Join("\n", vals));
    }
}

File: loj/3/solution.fs
open System
open System.IO

[<EntryPoint>]
let main _ =
    let outputPath = "copycat.out"
    if not (File.Exists("copycat.in")) then
        File.WriteAllText(outputPath, "")
    else
        let tokens =
            File.ReadAllText("copycat.in").Split(
                [| ' '; '\n'; '\r'; '\t' |],
                StringSplitOptions.RemoveEmptyEntries
            )

        if tokens.Length = 0 then
            File.WriteAllText(outputPath, "")
        else
            let t =
                match Int32.TryParse(tokens.[0]) with
                | true, v -> v
                | _ -> 0
            let vals = tokens |> Array.skip 1 |> Array.truncate t
            File.WriteAllText(outputPath, String.Join("\n", vals))
    0

